osu.Game/Screens/SelectV2/BeatmapCarouselFilterGrouping.cs

```diff
@@ -139,7 +139,7 @@ void addItem(CarouselItem i)
                     if (beatmap.StarRating > starGroup)
                     {
                         starGroup = (int)Math.Floor(beatmap.StarRating);
-                        return new GroupDefinition(starGroup + 1, $"{starGroup} - {starGroup + 1} *");
+                        return new GroupDefinition(starGroup + 1, string.Empty);
```

Comment:
I've removed the title here because it's not used in the star-diff group panel. I can bring it back, update it, and use it as a source for the star-diff group panel if preferred.

Reply:
I'd argue it's more correct to leave it for debug/correctness purposes.